### PR TITLE
add code and output samples for private tasks

### PR DESCRIPTION
**Pull request by sean-abbott**
Adds a little more flavor documentation to the private tasks explanations.

### CODE REVIEW

**Comment by sean-abbott** [comment]
addresses #173

File: doc/tasks.rst [hunk 1]
```diff
@@ -500,6 +500,27 @@ private/hidden tasks
...
 
+.. literalinclude:: tutorial/private_task.py
+
+.. code_block:: console
+
+    $ doit
+    .  hello
+    $ doit dumpdb
+    DBM type is 'dbm.gnu'
+    b'_private_get_username' -> {   '_values_:': {},
```
Comment (schettino72) [inline doc/tasks.rst]: oh. what an ugly output with those byte types `b'xxx'`

File: doc/tasks.rst [hunk 2]
```diff
@@ -500,6 +500,27 @@ private/hidden tasks
 
 If task name starts with an underscore '_', it will not be included in the output.
 
+.. literalinclude:: tutorial/private_task.py
+
+.. code_block:: console
+
+    $ doit
+    .  hello
+    $ doit dumpdb
```
Comment (schettino72) [inline doc/tasks.rst]: I do not think this is a good example. The output is very lengthy and the `dumpdb`  is an advanced command that most users would not know it exist or its purpose.

Better would be to have a private task that prints out something into the console. So you could see that it was executed although its name does not show up. 

what do you think?

**Comment by schettino72** [comment]
closing this as it seems inactive...

**Pull request closed without merging**